synapse/config/server.py

```diff
@@ -66,6 +129,10 @@
     "fe80::/10",
     # Unique local addresses.
     "fc00::/7",
+    # Testing networks.
+    "2001:db8::/32",
+    # Multicast.
+    "ff00::/8",
```

Comment:
Let's also add `fec0::/10:` Site Local Addresses (deprecated - RFC 3879), previously mentioned [here](https://github.com/matrix-org/synapse/pull/8870#discussion_r536253356).

```suggestion
    "ff00::/8",
    # Site-local addresses
    "fec0::/10",
```